components/tooltip/tooltip.js

```diff
@@ -148,6 +148,12 @@ class Tooltip extends RtlMixin(LitElement) {
 			 * @type {number}
 			 */
 			offset: { type: Number }, /* tooltipOffset */
+			/**
+			 * ADVANCED: Only show the tooltip if we detect the element of the id passed is truncating
+			 * If no id is set, the tooltip target element is used
+			 * @type {string}
+			 */
+			onlyShowIfTruncating: { type: String, attribute: 'only-show-if-truncating' },
```

Comment:
Are cases like the list item one that mean this can't just be a bool actually likely/planned to happen?

Reply:
Maybe not, I had it as a `boolean` when working with tags, and didn't realize it wouldn't work for list items until I was writing the visual-diffs. The main use cases I can think of (tags, buttons, and just general text in the immersive nav or wherever) should work fine with the boolean.  So we could always defer that, and add an additional `truncating-target` attribute if needed.